src/main/java/cam72cam/immersiverailroading/thirdparty/CommonAPI.java

```diff
@@ -50,26 +83,126 @@ public FluidStack getFluid() {
         return null;
     }
 
+    public Object[] getProperty(String PropertyName) {
+        return getProperty(StockProperty(PropertyName));
```

Comment:
This does not compile.

You will need to change the enum to have a getName and fromName method like:
https://onlinegdb.com/SkQDwmI_H